### PR TITLE
[UT] Add integration test for runtime profile

### PR DESCRIPTION
**Pull request by liuyehcf**
Fixes #issue

## What type of PR is this:
- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [x] UT
- [ ] Doc
- [ ] Tool

## Checklist:
- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5
  - [ ] 2.4

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

File: test/common/data/tpch/supplier.tbl
Comment (imay) [inline test/common/data/tpch/supplier.tbl:1]: Are these empty files neccessary?

Comment (liuyehcf) [reply test/common/data/tpch/supplier.tbl:1]: Yes, when downloading the real data into these directories, git will take a lot of time processing them, blocking all the subsequent commands that are executed inside the directory.

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Review by kangkaisen — APPROVED** [review]

**Review by andyziye — APPROVED** [review]

**Comment by liuyehcf** [comment]
@Mergifyio backport branch-3.1

**Pull request merged**